INSTALL.md

```diff
@@ -0,0 +1,81 @@
+# balena CLI Installation Instructions
+
+The easiest and recommended way of installing the CLI on all platforms (Linux, MacOS, Windows) is
+to use the [Standalone Installation](#standalone-installation) described below. Some specific CLI
+commands like `balena ssh` and `balena sync` have additional dependencies with separate
```

Comment:
Lets not mention `balena sync` this early on, especially since livepush is almost ready.